model/cartDao.js

```diff
@@ -0,0 +1,25 @@
+import prisma from '../prisma';
+
+const updateCartItemQuantity = async (userId, quantity, isTrue) => {
+  if (isTrue === true) {
+    await prisma.cart.update({
+      data: {
```

Comment:
```suggestion

const updateCartItemQuantity = async (userId, quantity, isTrue) => {
  if (isIncrement === true) {
    await prisma.cart.update({
      data: {
```

isTrue는 너무 의미가 부정확합니다

Reply:
네 ! 수정하겠습니다. 감사합니다 소헌님